environments/dataset_generator.py

```diff
@@ -52,7 +51,8 @@ def env_thread(args, thread_num, partition=True, use_ppo2=False):
         "record_data": not args.no_record_data,
         "multi_view": args.multi_view,
         "save_path": args.save_path,
-        "shape_reward": args.shape_reward
+        "shape_reward": args.shape_reward #,
+        #"env_rank": thread_num
```

Comment:
env_rank used at some point ?

Reply:
Oh yeah, it is not used right now, this is used for the first version of the multi-threading, but it is not used from the next version. So delete it.